src/common/subgraphMapper.ts

```diff
@@ -1,62 +1,40 @@
 import {
-  Account,
   Market,
   Order,
   Position,
   PriceFeedSnapshot,
   PythData,
-  Referral,
   Token,
-  Vault,
-  VaultCooldown,
-  VaultPosition,
+  Wallet,
 } from '../interfaces/subgraphTypes';
 
 ////////////////////////////////////////////////////////////////
 //////////////////////    ACCOUNT   ////////////////////////////
 ////////////////////////////////////////////////////////////////
 
-export const mapSubgraphResponseToAccountInterface = (response: any): Account | undefined => {
+export const mapSubgraphResponseToAccountInterface = (response: any): Wallet | undefined => {
   if (response === null) return undefined;
   try {
     return {
       id: response.id,
       totalOrdersCount: response.totalOrdersCount,
-      openPositionsCount: response.openPositionsCount,
       totalPositionsCount: response.totalPositionsCount,
-      totalReferralsCount: response.totalReferralsCount,
-      totalReferralRewardsInUsd: response.totalReferralRewardsInUsd,
-      unclaimedReferralRewardsWeth: response.unclaimedReferralRewardsWeth,
-      unclaimedReferralRewardsUsdc: response.unclaimedReferralRewardsUsdc,
       totalRealizedPnlPositions: response.totalRealizedPnlPositions,
-      totalRealizedPnlVaults: response.totalRealizedPnlVaults,
+      openPositionCount:response.openPositionCount,
       countProfitablePositions: response.countProfitablePositions,
       countLossPositions: response.countLossPositions,
       countLiquidatedPositions: response.countLiquidatedPositions,
       totalVolumeInUsd: response.totalVolumeInUsd,
       totalVolumeInUsdLongs: response.totalVolumeInUsdLongs,
       totalVolumeInUsdShorts: response.totalVolumeInUsdShorts,
       totalAccruedBorrowingFeesInUsd: response.totalAccruedBorrowingFeesInUsd,
-      totalStaked: response.totalStaked,
-      esPRFBalance: response.esPRFBalance,
     };
   } catch (error) {
     console.log('Error while mapping data', error);
     throw error;
   }
 };
 
-export const mapAccountsArrayToInterface = (response: any): Account[] | undefined => {
```

Comment:
This function will still be needed for mapping when multiple wallets are fetched in a single query